### DISCUSSION TITLE
4.2.0-alpha.0

### DISCUSSION THREAD

**Discussion by ronyeh** [0xfe/vexflow#1491] [General]
* Merge pull request #1480 from rvilarl/font/gonvilleSmufl_1 (d48f7491)
* Merge pull request #1490 from ronyeh/compare-images (35170273)
* Improve compare tool with more UI improvements to help with generating GIFs for reviewing PRs. (08191aad)
* gonville scaled (0fb14ef7)
* Update docs/api/ from recent PR merges. (86b8240d)
* Gonville glyphs generated from remap only SMuFL (ddd0e848)
* Merge pull request #1484 from ronyeh/compare-images (cfe6cb8d)
* Add test images. (b369ee93)
* Update image compare tool. (8abdadfc)
* Add some comments. (8c29e037)
* Show images side by side, or overlaid on top of each other. (2c4f8f8e)
* Image comparison tool so we can more easily review visual diffs between current and reference builds. (a0b04e96)
* Merge pull request #1487 from rvilarl/refactor/metricsStep (3e136294)
* stringnumber metrics (63998d22)
* Fix build errors. (b97e75b6)
* Allow developers to use https://pnpm.io/ without having to check in the lock file. (27f3652c)
* Merge pull request #1482 from rvilarl/fix/clefCodeAndTempoNotes (9f964a49)
* review comments (9eccfef4)
* binary visual regression, no threshold (bdec66a0)
* fix tempo notes (666aa084)
* fix percussion clef code (f4da8d6f)
* Remove build/ after releasing version 4.1.0 to npm and GitHub. (7067aefe)

<hr /><em>This discussion was created from the release <a href='https://github.com/0xfe/vexflow/releases/tag/4.2.0-alpha.0'>4.2.0-alpha.0</a>.</em>

**Comment by ronyeh** [comment]
Published to npm as 4.2.0-alpha.0.

https://www.npmjs.com/package/vexflow/v/4.2.0-alpha.0

@rvilarl @AaronDavidNewman @sschmidTU @mscuthbert 

Let us know if you find formatting issues or any bugs with this alpha release (Gonville SMuFL).

We determined that the diffs were minimal (except for bug fixes). If you do not find them acceptable, we can attempt to improve them so that they are closer to 4.1.0 Gonville.... or we can just bump the major revision to 5.0 before the release.

**Reply by sschmidTU** [reply]
Thanks for asking about our opinion, also in #1480!
I've been a bit busy with other projects, also within OSMD (1.7 release), and Vexflow 4+ integration in OSMD is a rather big project that has large visual diffs to analyze, so it's not so easy to determine the effect of a specific new change.

However, I'm very interested in integrating Vexflow 4 and checking out the new Gonville, so I'll try to test it out and share my view on it as soon as possible :)
(I have a very long train ride home tomorrow, that might be an opportunity, haha)
